pygfx/renderers/wgpu/wgsl/line.wgsl

```diff
@@ -97,14 +97,37 @@ fn rotate_vec2(v:vec2<f32>, angle:f32) -> vec2<f32> {
 
 struct VertexInput {
     @builtin(vertex_index) index : u32,
+    $$ if instanced
+    @builtin(instance_index) instance_index : u32,
+    $$ endif
+};
+
+$$ if instanced
+struct InstanceInfo {
+    transform: mat4x4<f32>,
+    id: u32,
 };
```

Comment:
As you said, I think we could e.g. move this to `std.wgsl`, but there's still the `instance_index` and `s_instance_infos` to account for. Plus it's not *that* much duplicate code, so let's leave it like this 👍